### DISCUSSION TITLE
Affichage cassé Bandeau

### DISCUSSION THREAD

**Discussion by Cnedellec** [AmauriC/tarteaucitron.js#755] [General]
Bonjour Amauri,

J'ai fait poser ce script juste après la balise head sur le site d'un annonceur (en preprod) : <script type="text/javascript" src="https://tarteaucitron.io/load.js?domain=preprod.thalasso.com&uuid=5d89345106348a46ebf837d706dbd1e505a9b674"></script>

Or au lieu d'avoir un affichage standard du bandeau, j'ai ceci dans le footer : 
![image](https://user-images.githubusercontent.com/86305646/124736544-66fc8880-df17-11eb-9b8d-70a755564e76.png)

A quoi cela est il dû ?

Merci de votre aide, 

Clémence

**Comment by AmauriC** [comment]
Ca ressemble à un conflict avec des règles CSS. Difficile d'aider plus sans avoir accès au site : /

**Comment by Cnedellec** [comment]
Je peux vous y donner accès, mais en privé si vous le souhaitez.

 

Merci, 

 

Clémence

 

De : Amauri CHAMPEAUX ***@***.***> 
Envoyé : mercredi 7 juillet 2021 11:56
À : AmauriC/tarteaucitron.js ***@***.***>
Cc : Cnedellec ***@***.***>; Author ***@***.***>
Objet : Re: [AmauriC/tarteaucitron.js] Affichage cassé Bandeau (#755)

 

Ca ressemble à un conflict avec des règles CSS. Difficile d'aider plus sans avoir accès au site : /

**Reply by AmauriC** [reply]
amauri@champeaux.fr ; )

**Comment by AmauriC** [comment]
> Refused to load the stylesheet 'https://tarteaucitron.io/cache/css/tarteaucitron.css?v=20210509' because it violates the following Content Security Policy directive: "style-src 'self' 'unsafe-inline' 'unsafe-eval' *.colissimo.fr *.mapbox.com *.gstatic.com *.cloudflare.com *.watcheezy.net *.googleapis.com *.jsdelivr.net *.payline.com *.popt.in". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.
